internal/pkg/cli/env_init.go

```diff
@@ -542,6 +548,24 @@ func (o *initEnvOpts) askAdjustResources() error {
 	return nil
 }
 
+func (o *initEnvOpts) validateDuplicateEnv() error {
+	envs, err := o.store.ListEnvironments(o.appName)
```

Comment:
Should we use `o.store.GetEnvironment()` instead? as it should be a faster endpoint